setup.py

```diff
@@ -1,15 +1,32 @@
+import pathlib
 import setuptools
 
 import lodge
 
-
 VERSION = "0.1.0"
```

Comment:
You should overwrite that version by the make publish command. Try using: `os.environ.get("VERSION", "0.1.0")

Reply:
I didn't get it. Why overwrite it when the developer is the responsible to update it? I think to do something like this it is need a command like `make release` where it could bump the version and create tags on git. But as we discussed in person, i don't think it is needed right now, since this project don't expect to much updates. 

The current flow to release it is to commit this version updated, release the tag and make publish with the version that you just created. I've set this variable `VERSION` to have intent on the version that you want to release. I could grep it from the setup.py but opted to leave like this. What you think @ryukinix?

Reply:
Oh! That's ok for now.